File: 43-reports/python-sdk/reports.py
import os, requests
API = "https://cost-katana-backend.store/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def generate_report(report_type, format):
    res = requests.post(f"{API}/reports/generate",
        json={"type": report_type, "format": format},
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"✅ Report generated: {res.json()['data']['id']}")
    return res.json()["data"]

File: 42-reports/npm-package/reports.ts
import axios from 'axios';
const API = 'https://api.costkatana.com/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function generateReport(type: string, format: string) {
  const res = await axios.post(`${API}/reports/generate`, {type, format},
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('✅ Report generated:', res.data.data.id);
  return res.data.data;
}
